examples/00-basic/13-manage_licensing.py

```diff
@@ -0,0 +1,85 @@
+"""
+.. _ref_manage_licensing:
```

Comment:
Hi @JennaPaikowsky, sorry for the late notice, I see that you were not added in the PR, would you mind giving us some feedback on this new example. @PProfizi, can you follow up on Jenna's comment please?